plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/metadata/collector/DeployVersionMetaDataCollectorTest.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.grandhackathon.deployment.sphere.plugin.metadata.collector;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import hudson.EnvVars;
+import hudson.model.TaskListener;
+import hudson.model.AbstractBuild;
+
+import java.io.PrintStream;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TreeMap;
+
+import lombok.extern.java.Log;
+
+import org.joda.time.DateTime;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.epam.grandhackathon.deployment.sphere.plugin.PluginJenkinsRule;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.Constants;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.model.DeploymentMetaData;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao.DeploymentMetaDataDao;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao.DeploymentMetaDataDaoMock;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao.EnvironmentDao;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao.EnvironmentDaoMock;
+import com.epam.grandhackathon.deployment.sphere.plugin.utils.DateFormatUtil;
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+
+@Log
+public class DeployVersionMetaDataCollectorTest {
+	
+	@Rule
+	public PluginJenkinsRule jenkinsRule = new PluginJenkinsRule();
+	
+	private static final Calendar DEPLOYMENT_DATE = new GregorianCalendar();
+	
+	private AbstractBuild<?, ?> initBuild(){
```

Comment:
private methods go after public and protected, set/get at the bottom of the class